index.tsx

```diff
@@ -510,6 +510,18 @@ class DraggableFlatList<T> extends React.Component<Props<T>, State> {
     this.flatlistRef.current._component.scrollToOffset({ offset })
   })
 
+  scrollToIndex = (index) => {
+    this.flatlistRef.current._component.scrollToIndex(index)
+  }
+
+  scrollToEnd = (params) => {
+		this.flatlistRef.current._component.scrollToEnd(params)
+  }	
+  
+  set getItemLayout(func) {
```

Comment:
why doesn't this work as a prop?

Reply:
I removed `set` keyword. Is there a better way to set `getItemLayout` property on FlatList or is it ok for you now?